src/dotnet/ShellShim/LinuxEnvironmentPath.cs

```diff
@@ -18,9 +18,9 @@ internal class LinuxEnvironmentPath : IEnvironmentPath
         private const string PathName = "PATH";
         private readonly BashPathUnderHomeDirectory _packageExecutablePath;
 
-        private readonly string _profiledDotnetCliToolsPath
-            = Environment.GetEnvironmentVariable("DOTNET_CLI_TEST_LINUX_PROFILED_PATH")
-              ?? @"/etc/profile.d/dotnet-cli-tools-bin-path.sh";
+        public static readonly string DotnetCliToolsProfilePath =
```

Comment:
why public?

Reply:
Will fix.  I didn't notice this type was public (assumed `internal`).

Reply:
Erm, wait this type is already internal.  I don't mind changing it to internal for consistency's sake.

Reply:
private readonly string _profiledDotnetCliToolsPath ? no? 

if not one is using it outside(And it should not. Other should use PathCalculator to get the path), let's make it private

Reply:
It's now being used by the tests to confirm that it created a (mock) file with the appropriate contents.

Reply:
i see, internal is fine